README.md

```diff
@@ -2,32 +2,43 @@ Parquet-cpp [![Build Status](https://travis-ci.org/apache/parquet-cpp.svg)](http
 ===========
 A C++ library to read parquet files.
 
-To build you will need some version of boost installed and thrift 0.7+ installed.  
-(If you are building thrift from source, you will need to set the THRIFT_HOME env
-variable to the directory containing include/ and lib/.)
-
-Then run:
-<br>
-<code>
-thirdparty/download_thirdparty.sh
-</code>
-<br>
-<code>
-thirdparty/build_thirdparty.sh
-</code>
-<br>
-<code>
-cmake . 
-</code>
-<br>
-<code>
-make
-</code>
-
-The binaries will be built to ./bin which contains the libraries to link against as
+## Third Party Dependencies
+- snappy
+- lz4
+- thrift 0.7+ [install instructions](https://thrift.apache.org/docs/install/)
+
+Many package managers support some or all of these dependencies. E.g.:
+```shell
+ubuntu$ sudo apt-get install libboost-dev libsnappy-dev liblz4-dev
+```
+```shell
+mac$ brew install boost snappy lz4 thrift
+```
+
+The thirdparty directory contains helper scripts to manually install third-party dependencies on Linux. You can use them like so.
+- `mkdir <build-dir>`
```

Comment:
I added the script and updated the readme.  Please have a look and give me your feedback.